### PR TITLE
Freemabd1/rw 467 Split out review and search yaml files

### PR DESCRIPTION
**Pull request by freemabd1**

### CODE REVIEW

File: api/build.rb [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
 def merge_yaml()
   Workbench::Swagger::merge_yaml('src/main/resources/workbench.yaml',
-    ['src/main/resources/client_api.yaml'], 'src/main/resources/merged.yaml')
+    ['src/main/resources/client_api.yaml', 'src/main/resources/cb_review_api.yaml', 'src/main/resources/cb_search_api.yaml'], 'src/main/resources/merged.yaml')
 end
 
 Common.register_command({
   :invocation => "merge-yaml",
-  :description => "Merges client_api.yaml and workbench.yaml together to produce a single YAML spec.",
+  :description => "Merges client_api.yaml, cb_review_api.yaml, cb_search_api.yaml and workbench.yaml together to produce a single YAML spec.",
```
Comment (danqrodney) [inline api/build.rb]: We can probably just say "Merges API YAML files together"

File: api/src/main/resources/client_api.yaml [hunk 2]
```diff
@@ -208,6 +208,7 @@ definitions:
           The operator to use when comparing values. Defaults to EQUAL. If the "in" operator is used,
           either values or valueNumbers should be populated.
         $ref: "#/definitions/Operator"
+        default: '='
```
Comment (danqrodney) [inline api/src/main/resources/client_api.yaml]: So actually, I think there's something to be said for *not* defaulting stuff like this here... because it makes it impossible(?) for us to distinguish between researchers saying "yes give me equals" and not specifying an operator at all.

(I'm also changing the names of these operators.)

So maybe revert this change?

Comment (freemabd1) [reply api/src/main/resources/client_api.yaml]: Oh.. oops.. I was playing around with the default to see what happened and forgot to remove this. Please ignore this. I will revert.

File: api/build.rb (hunk 1 above)
Comment (freemabd1) [reply api/build.rb]: Done.
